src/devices/Media/SoundDevice/SoundDevice.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+
+namespace Iot.Device.Media
+{
+    /// <summary>
+    /// The communications channel to a sound device.
+    /// </summary>
+    public abstract partial class SoundDevice : IDisposable
+    {
+        /// <summary>
```

Comment:
I think you're missing private parameterless constructor

Reply:
😅I don't quite understand why. It is a `abstract` class